src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Translations in Flex
+slug: translations
+updated: 2022-04-26
+category: concepts-development
+ingress:
+  This article introduces translations in Flex and how they are modified
+  and edited in Flex Console and the API
+published: true
+---
+
+In the FTW templates, user-facing content is not written directly into
+the source code. Instead, the source code uses the
+[ICU message syntax](https://unicode-org.github.io/icu/userguide/format_parse/messages/)
+that defines variables for each meaningful piece of content, and a
+translator or a content creator can define the specific words for the
+variable in their language. The end user only sees the translator's
+words, not the variable itself.
+
+```js
+  // ListingPage.js uses the variable to identify the message
+  const bookingSubTitle = intl.formatMessage({ id: 'ListingPage.bookingSubTitle' });
+...
+  // The wording of the message is defined separately
+  "ListingPage.bookingSubTitle": "Start by choosing your dates.",
+...
+  // This makes it possible to have several translations for a single variable,
+  // so that you can start developing your marketplace in any language, not just English
+  "ListingPage.bookingSubTitle": "Empieza escogiendo las fechas.",
+
+```
+
+Starting from XXXX-XX, marketplace operators can modify the wording of
+the translations in Flex Console. This means that operators can make
+changes to the marketplace texts without the need for code changes. In
+addition, the same translations can now be used from several different
+client applications, making it easier to make centralized changes.
+
+## How translations are handled in Flex
+
+- What is a marketplace asset?
+
+With Console-editable translations, Flex introduces the concept of
+[marketplace assets](/references/marketplace-assets/). Marketplace
+assets are ways to define marketplace content and configurations with
```

Comment:
"assets are ways" sounds a bit off. Assets are _things_, files, images, and so on. So they might _provide_ ways to define marketplace content and configs, but I wouldn't say they _are_ that.